PortfolioApp/UsersScreen/Users/UsersTableViewController.swift

```diff
@@ -0,0 +1,71 @@
+import UIKit
+
+final class UsersTableViewController: UITableViewController {
+    public typealias ReusableType = UsersTableViewController
+    @IBOutlet private weak var usersTableView: UITableView!
+    var presenter = UsersPresenter()
+    var models: [User] = []
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        presenter.view = self
+        tableView.refreshControl = UIRefreshControl()
```

Comment:
Maybe it is better to write all tables config in separate method, like `configureTableView`